packages/css-in-js-helpers/src/index.js

```diff
@@ -25,9 +25,73 @@ const toCSS = item => {
   )}: ${val.trim().replace(/'|,/g, '')};`
 }
 
+const firstCharsEqual = (str1, str2) => str1[0].toLowerCase() === str2[0].toLowerCase()
+
+const lineEndsWithComma = text => /,\s*$/.test(text)
+
+const isPropertyValuePrefix = prefix => prefix.trim().length > 0 && prefix.trim() !== ':'
+
+const firstInlinePropertyNameWithColonPattern = /(?:{{|{)\s*(\S+)\s*:/
+
+const inlinePropertyNameWithColonPattern = /(?:,.+?)*,\s*(\S+)\s*:/
+
+const inlinePropertyStartWithColonPattern = /(?::.+?)*,\s*/
+
+const propertyNameWithColonPattern = /^\s*(\S+)\s*:/
+
+const propertyNamePrefixPattern = /[a-zA-Z]+[-a-zA-Z]*$/
+
+const pseudoSelectorPrefixPattern = /\':(:)?([a-z]+[a-z-]*)?(\')?$/
+
+const tagSelectorPrefixPattern = /(^|\s|,)([a-z]+)?$/
+
+const importantPrefixPattern = /(![a-z]+)$/
+
+const cssDocsURL = 'https://developer.mozilla.org/en-US/docs/Web/CSS'
+
+const rnDocsURL = 'https://facebook.github.io/react-native/docs'
+
+function __guard__(value, transform) {
+  return typeof value !== 'undefined' && value !== null
+    ? transform(value)
+    : undefined
+}
+
+function isPropertyNamePrefix(prefix) {
+  if (prefix == null) {
+    return false
+  }
+  prefix = prefix.trim()
+  return prefix.length > 0 && prefix.match(/^[a-zA-Z-]+$/)
+}
+
+function getImportantPrefix(text) {
+  return __guard__(importantPrefixPattern.exec(text), x => x[1])
+}
+
 export default function convert(s) {
   const lines = s.split('\n')
   return lines
     .map(item => (isCSS(item) ? toJS(item) : isJS(item) ? toCSS(item) : item))
     .join('\n')
 }
+
+export {
```

Comment:
@ansumanshah many of these functions could be used in the atom version and dry up the codebase.

See: https://github.com/ansumanshah/css-in-js/blob/fa7b78d/packages/atom-css-in-js/lib/autocomplete.js#L6-L19

Reply:
@paulmolluzzo true, will work on moving this as soon as we have the first alpha release for vscode ready 🎉